gmsh_utils/gmsh_IO.py

```diff
@@ -640,12 +640,12 @@ def extract_geo_data(self):
             name = gmsh.model.getPhysicalName(group[0], group[1])
 
             # gets entity per group
-            entity = gmsh.model.getEntitiesForPhysicalGroup(group[0], group[1])[0]
+            entities = gmsh.model.getEntitiesForPhysicalGroup(group[0], group[1])
 
             # add group to dictionary
             geo_data["physical_groups"][name] = {"ndim": group[0],
                                                  "id": group[1],
-                                                 "geometry_id": entity}
+                                                 "geometry_ids": list(entities)}
```

Comment:
why not return a list from here:
gmsh.model.getEntitiesForPhysicalGroup(group[0], group[1])